apps/console/src/features/validation/api/validation-config.ts

```diff
@@ -116,7 +125,11 @@ export const useValidationConfigData = <Data = ValidationDataInterface[], Error
     };
 };
 
-const prepareValidationConfigData = (values: ValidationFormInterface): ValidationDataInterface => {
+const preparePasswordValidationConfigData = (values: ValidationFormInterface): ValidationDataInterface => {
+
```

Comment:
Shall we remove this empty line?

Reply:
Discussed to keep this line for better readability.